src/Raven.Server/Documents/DocumentConflict.cs

```diff
@@ -10,5 +10,30 @@ public class DocumentConflict
         public BlittableJsonReaderObject Doc;
         public long StorageId;
         public ChangeVectorEntry[] ChangeVector;
+
```

Comment:
It is better to be explicit about it, otherwise you are going to have very hard to read code.
Change it to a factory method and call that.